### ISSUE TITLE
[Feature Request] Implement Landing Page

### ISSUE THREAD

**Issue by samiurprapon** [samiurprapon/Aether#189]
### 🚀 Feature Request

**Description:**
We are looking to create a new **Landing Page** using **React**. This landing page will serve as the first point of contact for users, showcasing our product and encouraging user engagement.

## 📂 Scope

- [ ] 📱 Android
- [ ] 🖥️ Backend
- [X] 🖼️ Frontend
- [ ] 📝 Documentation

### 🎨 Design & Mockup:

![LandingPage](https://github.com/user-attachments/assets/c3e43d67-7925-43d2-9120-df337d8ba20d)

### 📋 Requirements:
1. **Light/Dark Mode**

### 🛠️ Tech Stack:
- React
- TypeScript

### 📝 Additional Context:
- You can offer different designs but keep this `eye image`

**Comment by mayank-0789** [comment]
Hey @samiurprapon  I would like to work upon this issue

**Comment by Che140401** [comment]
Hey @samiurprapon,

I’ve worked on multiple projects as a MERN stack engineer, specializing in frontend development. I have experience maintaining clean code and contributing to projects following best practices.

Additionally, I recently participated in the GirlScriptCode Hackathon, which further honed my skills in collaborative development.

I'd love to contribute to this project. Please let me know how I can assist!

[Here’s a link](https://portfolio-che140401s-projects.vercel.app/) to my portfolio for reference.

**Comment by samiurprapon** [comment]
Hi @Che140401 , I have assigned you to this issue. 

Fork this repository and follow this guide - https://github.com/samiurprapon/Aether/blob/main/CONTRIBUTING.md

**Comment by samiurprapon** [comment]
@mayank-0789  you can create the Authentication pages if you want, I can assign you to do that, here you can get the figma design 
- https://www.figma.com/design/07DX15asGMIA8VWwTqZYUl/aether?node-id=0-1&t=FS5zurlfhYIFwH9W-1

**Comment by mayank-0789** [comment]
Hey @samiurprapon yes please assign it to me

**Comment by TheFastest599** [comment]
Hey @samiurprapon ,
I can make this website landing page , but will be using React and Javascript and maybe Tailwind CSS.
Will it do?
My portfolio - [https://anirban-saha-portfolio.netlify.app](url)

**Comment by samiurprapon** [comment]
@TheFastest599 Yes you can do it. Currently, We need a UX designer to Implement more pages to work on. I will keep you updated today.

**Comment by Che140401** [comment]
hey @samiurprapon I have created my PR req so do merge it in your code.

**Comment by samiurprapon** [comment]
@mayank-0789 any update from your side ?

**Comment by samiurprapon** [comment]
> hey @samiurprapon I have created my PR req so do merge it in your code.

Hey I asked for few changes their.
